### PR TITLE
feat: add star documentation proposal

### PR DESCRIPTION
**Pull request by coyotespike**
This PR follows (I hope!) the new process for a grant proposal. I understand this is an evolving process, so please feel free to suggest changes to this proposal.

### CODE REVIEW

File: content/grants/star-documentation-proposal.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+title = "How to Buy a Star: a Guide"
+date = 2021-06-08
+[taxonomies]
+grant_type = ["proposals"]
+grant_category = ["Documentation"]
+[extra]
+image = ""
+description = "Create a guide with helpful vocabulary and essential how-tos"
+reward = 1
+assignee = "@sonnep-ribbep"
```
Comment (jalehman) [inline content/grants/star-documentation-proposal.md]:
```suggestion
assignee = "~sonnep-ribbep"
```

**Review by jalehman — APPROVED** [review]
One change, and then LGTM

**Comment by mikeosborne** [comment]
Please change location from content/grants to content/grants/proposals, as per the grants reorg PR #639

**Comment by coyotespike** [comment]
@mikeosborne Thanks for that catch, updated!

**Comment by jalehman** [comment]
@mikeosborne can you figure out why this guide doesn't show up in the main grants list?
Screen Shot 2021-06-14 at 8 52 46 AM

**Comment by mikeosborne** [comment]
When I pull the changes down to my local copy, the proposal shows up in the main grants list. The only thing I can figure out is coyotespike's forked repo isn't using the updates from the grants reorg PR. I'm not sure how to correct this, but I believe he should pull from the current master then apply his changes.

**Comment by jalehman** [comment]
Ah, I see. If you verified it with the latest changes in master already applied then we should be good to go on this, since the PR will merge into master.

**Pull request merged**